devicetypes/iblinds/iblinds-zwave.src/iblinds-zwave.groovy

```diff
@@ -0,0 +1,299 @@
+/**
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+
+metadata {
+    definition (name: "iblinds Z-Wave", namespace: "iblinds", author: "HABHomeIntel", ocfDeviceType: "oic.d.blind",  mnmn: "SmartThings", vid: "generic-shade-3") {
+
+        capability "Switch Level" 
+        capability "Switch" 
+        capability "Battery" 
+        capability "Refresh" 
+        capability "Actuator"
+        capability "Health Check" 
+        capability "Window Shade" 
+        capability "Window Shade Preset"	
+
+        command "stop"
+
+
+        fingerprint mfr:"0287", prod:"0003", model:"000D", deviceJoinName: "iBlinds Motor"
```

Comment:
According to new fingerprint writing rules - deviceJoinName should be changed to "iBlinds Window Treatment"